### ISSUE TITLE
Monitor whether Dependabot properly bumps actions in `dependabot/.github/workflows`

### ISSUE THREAD

**Issue by blakeNaccarato** [blakeNaccarato/copier-python#29]
These files technically have a broken `python-version` field since they're meant to be suffixed by `.jinja`.

**Comment by blakeNaccarato** [comment]
Okay, a number of things were implemented to fix this. It was not done in a Pull Request because I had to trigger Dependabot multiple times and try different behaviors to get it to work. Here is a summary of what fixed it:
- Put quotes around `{% raw %}${{ secrets.PYPI_TOKEN }}{% endraw %}` so it can be parsed at the YAML stage but is also valid Jinja.
- Put quotes around Jinja flags such as `{{ python_version }}` so it can be parsed at the YAML stage.
- A user with push access must manually merge the Pull Request made by Dependabot, instead of commanding `@dependabot merge`. This is so that the auto-commit action in `bump` will have a sufficiently-privileged co-author.

In conclusion, we cannot use `@dependabot merge` when we want `bump` to run after the merge, due to authentication issues. A workaround would be creating a personal user account intended as a bot, and giving it push access to this repo. But I think that simpler is better, and explicitly elevating the privileges by manually merging seems the better option.
